editor/src/utils/clipboard.ts

```diff
@@ -150,28 +160,39 @@ export function createDirectInsertImageActions(
 
 export function createClipboardDataFromSelectionNewWorld(
```

Comment:
Might as well rename this to remove the `NewWorld` whilst you're here (assuming the old world version no longer exists)

Reply:
fixed